.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       run: gem install bundler && bundle install --jobs 4 --retry 3
 
     - name: Run test suite
-      run: bin/rake test
```

Comment:
Does this not work with 2.7?

Reply:
Yeah it's a bug in the bundler embeded with 2.7. The same bug that makes nested `bundle exec` fail.